.github/workflows/setup-rust-cache/action.yaml

```diff
@@ -0,0 +1,9 @@
+name: Setup Rust cache
```

Comment:
nit: if this sub-workflow is as simple as just one step of two lines, maybe we should inline it? What do you think about it

Reply:
agree, inline is clear enough.